<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>162</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="saleProfit">
     <property name="prefix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <double>0.020000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="button">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>卖出股票后统一扣除交易手续费</string>
     </property>
     <property name="text">
      <string>交易手续费</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="cost">
     <property name="prefix">
      <string notr="true"/>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.002000000000000</double>
     </property>
     <property name="maximum">
      <double>0.008000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="firstBuyPercent">
     <property name="prefix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <double>0.020000000000000</double>
     </property>
     <property name="maximum">
      <double>0.300000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="decreaseBuy">
     <property name="prefix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <double>0.020000000000000</double>
     </property>
     <property name="maximum">
      <double>0.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>加仓跌幅</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>单次买入百分比</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>止盈</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
